src/scripts/farming/Farming.ts

```diff
@@ -1233,8 +1233,9 @@ class Farming implements Feature {
         }
     }
 
+    public static genBounds = [0, 8, 20, 35, 53, Infinity];
     public static getGeneration(gen: number): BerryType[] {
-        const genBounds = [0, 8, 20, 35, 53, Infinity];
+        const genBounds = Farming.genBounds;
         return App.game.farming.berryData.filter(berry => berry.type >= genBounds[gen] && berry.type < genBounds[gen + 1]).map(berry => berry.type);
```

Comment:
For the above change to work:
```suggestion
        const minBound = genBounds[gen - 1] || 0;
        const maxBound = genBounds[gen] || Infinity;
        return App.game.farming.berryData.filter(berry => berry.type >= minBound && berry.type < maxBound).map(berry => berry.type);
```